pkg/conversion/converter.go

```diff
@@ -670,6 +670,32 @@ func (c *Converter) defaultConvert(sv, dv reflect.Value, scope *scope) error {
 		switch st.Kind() {
 		case reflect.Ptr, reflect.Interface:
 			return c.convert(sv.Elem(), dv.Elem(), scope)
+
+		case reflect.Slice:
```

Comment:
A conversion from string slice to unversioned.Time is sufficient (e.g. https://github.com/liggitt/kubernetes/commit/a509096af1bf172cbf2460db7333d766f670821a#diff-380d3fffd8583e3e031622a90dcaec36)